configmaptocrs/Dockerfile

```diff
@@ -16,9 +16,22 @@ COPY api api
 # COPY configmaptocrs
 COPY configmaptocrs/*.go configmaptocrs/
 
+ARG TARGETARCH
+ARG TARGETOS
+ARG TARGETPLATFORM
```

Comment:
Do these need to be passed in the GH Actions config?

Reply:
They are automatically populated by docker buildx